luaui/Widgets/gfx_DrawUnitShape_GL4.lua

```diff
@@ -212,30 +212,50 @@ local VBOTables = {}
 local corUnitDefIDs = {}
 local armUnitDefIDs = {}
 
+local owners = {} -- maps uniqueIDs to their optional owners
 
 local uniqueID = 0
 
-local function DrawUnitGL4(unitID, unitDefID, px, py, pz, rotationY, alpha, teamID, teamcoloroverride, highlight)
-	-- Documentation for DrawUnitGL4:
-	--	unitID: the actual unitID that you want to draw
-	--	unitDefID: which unitDef is it (leave nil for autocomplete)
-	-- px, py, py: Apply an offset to the position of the unit, usually all 0
-	-- rotationY: Angle in radians on how much to rotate the unit around Y, usually 0
-	-- alpha: the transparency level of the unit
-	-- teamID: which teams teamcolor should this unit get, leave nil if you want to keep the original teamID
-	-- teamcoloroverride: much we should mix the teamcolor into the model color [0-1]
-	-- highlight: how much we should add a highlighting animation to the unit (blends white with [0-1])
-	-- returns: a unique handler ID number that you should store and call StopDrawUnitGL4(uniqueID) with to stop drawing it
-	-- note that widgets are responsible for stopping the drawing of every unit that they submit!
+local instanceCache = {}
+for i= 1, 14 do instanceCache[i] = 0 end
+
+
+---DrawUnitGL4(unitID, unitDefID, px, py, pz, rotationY, alpha, teamID, teamcoloroverride, highlight, updateID, ownerID)
+---note that widgets are responsible for stopping the drawing of every unit that they submit! They may use RemoveMyDraws(ownerID) 
+---@param unitID number the actual unitID that you want to draw
+---@param unitDefID number which unitDef do you want to draw
+---@param px number optional where in the world to do you want to draw it
+---@param py number optional where in the world to do you want to draw it
+---@param pz number optional where in the world to do you want to draw it
+---@param rotationY number optional Angle in radians on how much to rotate the unit around Y, 0 means it faces south, (+Z), pi/2 points west (-X) -pi/2 points east
+---@param alpha number optional the transparency level of the unit
+---@param teamID number optional which teams teamcolor should this unit get, leave nil if you want to keep the original teamID
+---@param teamcoloroverride optional number much we should mix the teamcolor into the model color [0-1]
+---@param highlight number optional how much we should add a highlighting animation to the unit (blends white with [0-1])
+---@param updateID number optional specify the previous uniqueID if you want to update it
+---@param ownerID any optional unique identifier so that widgets can batch remove all of their own stuff
+---@return uniqueID number a unique handler ID number that you should store and call StopDrawUnitGL4(uniqueID) with to stop drawing it
+local function DrawUnitGL4(unitID, unitDefID, px, py, pz, rotationY, alpha, teamID, teamcoloroverride, highlight, updateID, ownerID)
```

Comment:
Same points on the docs as from `DrawUnitShapeGL4`. It's not clear based on the docs here what the difference between `DrawUnitGL4` and `DrawUnitShapeGL4` is, and when each should be used.